features/lexicon/package-lock.json

```diff
@@ -12049,19 +12568,21 @@
           "dev": true
         },
         "body-parser": {
-          "version": "1.19.0",
+          "version": "1.20.0",
```

Comment:
There are large changes in these `package-lock.json` files, most of which seem unrelated. Some of them are apparently due to the changes in `package.json` brought by the separation of the analysis and import modules; in that case those changes should have been carried out in those PRs. Can you please clarify the origin of these changes?